### PR TITLE
Add stylelint-disable-line command; closes #763

### LINKED ISSUE #763: Add support for disabling linting for single line (with inline comment)
I think it would be great if this could work in e.g. SCSS:

```css
selector {
  property1: value1;
  property2: value2; // stylelint-disable [rules]
  property3: value3;
}
```

Where stylelint would only be disabled for property2. 
As it is currently, this would disable all linting in the rest of the file if no `// style-lint enable` comment is found.

### PR DESCRIPTION
**Pull request by davidtheclark**

### CODE REVIEW

File: docs/user-guide/configuration.md [hunk 1]
```diff
@@ -203,6 +203,14 @@ Or you can turn off individual rules:
 /* stylelint-enable */
 ```
 
+And you can turn off rules for individual lines only, after which you do not need to explicitly re-enable them:
+
+```css
+#id { /* stylelint-disable-line */
+  color: pink !important; /* declaration-no-important */
```
Comment (jeddy3) [inline docs/user-guide/configuration.md]: Should this be `stylelint-disable-line declaration-no-important`

**Comment by jeddy3** [comment]
Nice. Thanks for doing this one!

I was also thinking a unique command (`stylelint-disable-line`) would probably be the best way to go about this.

In the [original issue](https://github.com/stylelint/stylelint/issues/763) there is a request for supporting `// ` disable comments. I think it's perfectly valid that standard css comments are required for these stylelint commands, and I think, as you said, less little need to add more to the codebase for a little sugar. How do you feel about that?

**Comment by davidtheclark** [comment]
I fixed the mistake in the docs. Good catch.

`//` comments should work just fine because postcss-scss parses them as comments. I added a test to demonstrate.

(`//` comments should only raise challenges when we target comments by some means other than PostCSS's parsed AST, like string searches that hunt for `/*`. Because when we are actually looking for comment nodes, those could have originated in regular or `//` comments -- either way.)

**Comment by jeddy3** [comment]
> (// comments should only raise challenges when we target comments by some means other than PostCSS's parsed AST, like string searches that hunt for /*.

Understood. I thought as much, but I wasn't entirely sure. Thanks for adding the test in to demonstrate :)

This looks great to me now. Thanks again.

**Pull request merged**